bin/find-flakey-browser-tests

```diff
@@ -28,36 +24,39 @@ Help()
   echo
 }
 
-NUMRUNS=1
-TS_TEST=""
+number_of_runs=1
+ts_test=""
 while getopts n:t:h flag; do
   case "${flag}" in
-    n) NUMRUNS=${OPTARG}
+    n)
+      number_of_runs="${OPTARG}"
+      ;;
+    t)
+      ts_test="${OPTARG}"
       ;;
-    t) TS_TEST=${OPTARG}
+    h)
+      print_help
+      exit 0
       ;;
-    h) Help
-      exit;;
   esac
 done
+
 shift "$(( OPTIND - 1 ))"
 
 COMMIT=$(git rev-parse HEAD)
-FLAKESDIR="flakes/$COMMIT"
-mkdir -p $FLAKESDIR
+FLAKESDIR="flakes/${COMMIT}"
+mkdir -p "${FLAKESDIR}"
 
-echo "Finding flakes for commit $COMMIT"
-for i in $( seq 1 $NUMRUNS )
+echo "Finding flakes for commit ${COMMIT}"
+for i in $( seq 1 "${number_of_runs}" )
 do
-  printf "Starting browser test run #$i..."
-  bin/run-browser-tests ${TS_TEST} > $FLAKESDIR/tmp 2>&1
+  printf "Starting browser test run #${i}..."
+  bin/run-browser-tests "${ts_test}" > "${FLAKESDIR}/tmp" 2>&1
   if [ $? -eq 0 ]; then
     printf "PASS\n"
```

Comment:
there's a couple printfs in here that may as well be echos, and the majority of the code uses echo

Reply:
Yeah, printf does have a difference in behavior in that it will interpret "\n" as a newline rather than print the literal chars. I changed the code to use echo twice instead and use `out::error` for the failure message

Reply:
echo ends with a newline by default so you don't need it, but `echo -e` will process escape sequences like `echo -e "first\nsecond"